powers.c

```diff
@@ -0,0 +1,30 @@
+/*Ashank Kumar, prints powers of 2 less then inputted num*/
+/*Add check for wrong input*/
+#include <stdio.h>
+#include <math.h>
+
+int main()
```

Comment:
you misunderstood the assignment. the number being input is not n - the number being inputted is the number that 2^n should be less than. Also, you didn't print 2^0 = 1.